managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/MetaStoreImpl.java

```diff
@@ -56,29 +58,39 @@
     private final OrderedExecutor executor;
 
     private static final int MAGIC_MANAGED_LEDGER_INFO_METADATA = 0x4778; // 0100 0111 0111 1000
-    private final CompressionType compressionType;
+    private static final int MAGIC_MANAGED_CURSOR_INFO_METADATA = 0x4778; // 0100 0111 0111 1000
```

Comment:
Why do we define the same static immutable value?

Reply:
I consider using the same MAGIC number,  want to see your feedback.